msvc/libmonoutils.vcxproj

```diff
@@ -42,6 +42,9 @@
     <ClCompile Include="..\mono\utils\mono-io-portability.c" />
     <ClCompile Include="..\mono\utils\mono-linked-list-set.c" />
     <ClCompile Include="..\mono\utils\mono-logger.c" />
+    <ClCompile Include="..\mono\utils\mono-log-windows.c" />
+    <ClCompile Include="..\mono\utils\mono-log-common.c" />
+    <ClCompile Include="..\mono\utils\mono-log-posix.c" />

```

Comment:
mono-log-posix.c won't be used on windows so if its added into vcxproj file it should be excluded from build or it could generate empty object file triggering unnecessary build warnings.